Notification.Application/Commands/Mail/MailCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿using MailKit.Net.Smtp;
+using MediatR;
+using Microsoft.Extensions.Options;
+using MimeKit;
+using Notification.Application.Handler;
+using Notification.Application.Logger;
+using Notification.Contracts.Settings.Mail;
+using Notification.Data.Entities;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Notification.Application.Commands.Mail
+{
+    public class MailCommand : IRequest
+    {
+        public MailCommand(User user, Book book)
+        {
```

Comment:
Specify somehow the name. 
It's just don't clear for now. Because it's only from code clear that we need to tell user about available book